pom.xml

```diff
@@ -56,6 +57,11 @@
                 <artifactId>siddhi-annotations</artifactId>
                 <version>${siddhi.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.siddhi.extension.map.json</groupId>
+                <artifactId>siddhi-map-json</artifactId>
+                <version>${siddhi.map.json.version}</version>
+            </dependency>
```

Comment:
You need above extension only for tests right? Then add `Test` scope for it.

Reply:
Fixed in https://github.com/siddhi-io/siddhi-store-elasticsearch/pull/41/commits/d19e2e20f87be7d4a4772619796a5dfbc40bd36b commit